azure-fs-aks/README.md

```diff
@@ -0,0 +1,49 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure Kubernetes Service (AKS) Cluster
+
+Stands up an [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS) cluster.
+
+## Prerequisite
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+2. [Install .NET Core 3.1+](https://dotnet.microsoft.com/download)
+3. [Install Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
```

Comment:
Either point to `make` as a required tool or suggest an alternative.
Does `make` work on Windows?

Reply:
I'll add an extra section describing that you can just run `pulumi up --yes` and all the other things.